docs/actions/refresh-grid.md

```diff
@@ -0,0 +1,43 @@
+---
+id: refresh-grid
+title: Refresh Action Grid
+sidebar_label: Refresh Action Grid
+---
+
+> Audience: [`Citizen Developers`](/docs/audience#citizen-developers)<br/>
+> Skill Prerequisites: `Using Action Grid`
+
+This action is usually used to refresh the grid after executing another action.
+
+## `Typical Use Cases`
+
+- Refresh Action Grid with updated without reloading the entire page
+
+## `Actions Covered by this Document`
+
+| Action | Purpose |
+| -- | -- |
+| Refresh Action Grid | Refresh an Action Grid module without reloading the page |
+
+## `Input Parameter Reference`
+
+| Parameter | Description | Supports Tokens | Default | Required |
+| -- | -- | -- | -- | -- |
+| Select Module | Select the Grid Module you wish to refresh| Yes | Empty String | No |
+
+## `Javascript method`
+
+Using the javascript API you can refresh the grid by calling the javascript method
+
+`dnnsf.api.actionGrid.refresh(1234)`
+
+First parameter is required and is the module id of the Action Grid to be refreshed
+
+## `Related Actions`
+
+| Action Name | Description|
```

Comment:
When I first came on board, I went through the site and removed a bunch of these links that were broken. Unless we had these docs soon, I think we should keep this section, but just leave the action names unlinked for now. Later, we can create a separate PR that goes through docs and adds links to other actions.